src/OSPSuite.Core/Domain/Data/QuantityValues.cs

```diff
@@ -47,6 +49,12 @@ public virtual float[] Values
          set => _values = value ?? new float[] {};
       }
 
+      public virtual double[] Sensitivities
+      {
+         get => _sensitivities;
+         set => _sensitivities = value ?? new double[] {};
```

Comment:
Why would this be set to null?

Reply:
nevermnd. This would be good for serialization so that we do not serialize empty entries